### DISCUSSION TITLE
Block Shorts from search results?

### DISCUSSION THREAD

**Discussion by HubKing** [FreeTubeApp/FreeTube#4656] [Q&A]
99% of "Shorts" are garbage. Can I block all Shorts? 

In desktop browsers, I use https://github.com/gijsdev/ublock-hide-yt-shorts

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
With the current information provided by YouTube, it's impossible to hide shorts on all pages, the only way we know something is a short, is if it's on a shorts specific page.

**Reply by HubKing** [reply]
Can't it be locally filtered? I mean, the current length options of the search (<4, 4~20, 20<) are from YouTube's server, but the app could have an additional local filter option for search like "Exclude length <1", and locally remove videos that are shorter than 1 minute from the search result received from YouTube.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
There are allot of vids that arent shorts that would be caught in the crossfire here so no...

**Reply by Hestichan** [reply]
97% of videos under 1 min are shorts so YES. filtering by time is absolutely a need to add feature!

**Reply by ZTetriminos** [reply]
Honestly, the search function is unusable for me due to the flood of shorts in the results. I have to remember the exact name of the video (except for music videos), otherwise I'd have to scroll past 50 unrelated shorts. I'd take any solution. A custom filter for video length seems the best option.
